src/js/pages/Tickets.js

```diff
@@ -0,0 +1,55 @@
+import {$} from "../modules/utils.js";
+import Component from "../lib/Component.js";
+
+export default class Tickets extends Component {
+  setup() {
+    this.onToggle = this.onToggle.bind(this);
+  }
+
+  template() {
+    const {lottoNumbers} = this.props;
+    const eleLottoTicket = '<span class="mx-1 text-4xl">🎟️ </span>';
+    const eleLottoTicketAndNumbers = numbers => `<div class="mx-1 text-4xl d-flex items-center">🎟️ <span class="mx-3 text-xl">${numbers.join(', ')}</span></div>`;
+
+    return `
+      <section class="mt-9">
+        <div class="d-flex">
+          <label class="flex-auto my-0">총 <span id="purchaseAmount">${lottoNumbers.length}</span>개를 구매하였습니다.</label>
+          <div class="flex-auto d-flex justify-end pr-1">
+            <label class="switch">
+              <input type="checkbox" class="lotto-numbers-toggle-button""/>
+              <span class="text-base font-normal">번호보기</span>
+            </label>
+          </div>
+        </div>
+        <div id="lottoTickets" class="ticket d-flex flex-wrap show">
+            ${lottoNumbers.reduce((lottos, lottoNumber) => {
```

Comment:
```suggestion
            ${lottoNumbers.reduce((lottos) => {
```
사용하지 않으니 지워주시면 될것 같네요